### DISCUSSION TITLE
Which interface type to use for telemetry equipment.

### DISCUSSION THREAD

**Discussion by EirikMaarstad** [netbox-community/netbox#10248] [Q&A]
Hi! 

Before sending in a feature request, I wanted to ask the community what type of port they would use for BNC coax cables that connect to telemetry devices. 

The ports in question would classify as interfaces, as they are used to exchange data between devices in this system. 
However, BNC (nor SMA) exist as interface types that can be used. Front and Rear ports do have BNC as a type option, however only a fraction of the devices have pass-through ports, and I'd end up with multiple rear ports that could be mapped to front ports that will never exist. 

Anyone else using Netbox as a source of truth for telemetry systems, or other systems that utilize coax cables quite heavily?
Any input welcome!

**Comment by candlerb** [comment]
We have satellite feeds coming in on BNCs, and although I've not got round to documenting them, I would just create interfaces of type "Other".

**Reply by EirikMaarstad** [reply]
Thank you for the reply. 

I've opted to use Rear ports after all. The end result, in terms of cabletracing remains the same, and I've hidden the Positions column from the overview.
